### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by scottdangelo**
TODO: add sample output and arch diagram

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,97 @@
...
+
+* An [IBM Cloud Account](https://console.bluemix.net).
+* [IBM Cloud CLI](https://console.bluemix.net/docs/cli/index.html#overview)
+* An account on [AWS SageMaker](https://aws.amazon.com/sagemaker/)
+
+# Steps
+
+1. [Clone the repository](#1-clone-the-repository)
+1. [Create a Compose for PostgreSQL DB](#2-create-a-compose-for-postgresql-db)
+1. [Create an AI OpenScale service](#3-create-an-aiopenscale-service)
```
Comment (markstur) [inline README.md]: there is a step 5 (somehow no step 4 -- because it is bad luck?)

File: README.md [hunk 2]
```diff
@@ -1 +1,97 @@
...
+
+# Monitor Sagemaker ML With AI OpenScale
+
+In this Code Pattern, we will use data from the [UC Irvine Machine Learning database](https://archive.ics.uci.edu/ml/machine-learning-databases/breast-cancer-wisconsin/) to create a logistic regression model using Amazon SageMaker. We will use AI OpenScale to bind the ML model deployed in the AWS cloud, create a subscription, and perform payload and feedback logging.
+
+When the reader has completed this Code Pattern, they will understand how to:
+
+* Prepare data, train a model, and deploy using AWS SageMaker.
+* Score the model using sample scoring records and the scoring endpoint.
+* Setup AI OpenScale Data Mart
```
Comment (markstur) [inline README.md]: nit:  mix of using periods and not using periods

File: README.md [hunk 3]
```diff
@@ -1 +1,97 @@
...
+
+* Wait a couple of minutes for the database to be provisioned.
+* Click on the `Service Credentials` tab on the left and then click `New credential +` to create the service credentials. Copy them or leave the tab open to use later in the notebook.
+
+### 3. Create an AI OpenScale service
+
+* Using the [IBM Cloud Dashboard]() create an [AI OpenScale](https://console.bluemix.net/catalog/services/ai-openscale) service.
+* You will get the AI OpenScale instance GUID when you run the notebook using the [IBM Cloud CLI](https://console.bluemix.net/catalog/services/ai-openscale)
+
+### 5. Run the notebooks
```
Comment (markstur) [inline README.md]: skipped step 4

File: README.md [hunk 4]
```diff
@@ -1 +1,97 @@
...
+* Move your cursor to each code cell and run the code in it. Read the comments for each cell to understand what the code is doing. **Important** when the code in a cell is still running, the label to the left changes to **In [\*]**:.
+  Do **not** continue to the next cell until the code is finished running.
+
+#### Next Run [AIopenScaleSageMakerMLengine.ipynb](notebooks/AIopenScaleSageMakerMLengine.ipynb)
+
+* Follow the instructions for `ACTION: Get AI OpenScale instance_guid and apikey` using the [IBM Cloud CLI](https://console.bluemix.net/docs/cli/index.html#overview)
+
+How to get api key using ibmcloud console:
+```bash
+$ibmcloud login --sso
```
Comment (markstur) [inline README.md]: remove the $ signs

File: README.md [hunk 5]
```diff
@@ -1 +1,97 @@
...
+
+* Enter the `instance_guid` and `apikey` in the next cell for the `AIOS_CREDENTIALS`.
+* In the cell after that enter `POSTGRES_CREDENTIALS` using the value for the PostreSQL credentials from [Step #2](#2-create-a-compose-for-postgresql-db).
+* In the cell after `2.1 Bind SageMaker machine learning engine` enter your `SAGEMAKER_ENGINE_CREDENTIALS`.
+* After running `3.1 Add subscriptions` you will get a `source_uid` to enter in the cell that follows.
+* Move your cursor to each code cell and run the code in it. Read the comments for each cell to understand what the code is doing. **Important** when the code in a cell is still running, the label to the left changes to **In [\*]**:.
+  Do **not** continue to the next cell until the code is finished running.
+
+# Sample Output
+
```
Comment (markstur) [inline README.md]: shoot, I was looking for the sample output but there ain't none

File: README.md [hunk 6]
```diff
@@ -1 +1,97 @@
...
+* Prepare data, train a model, and deploy using AWS SageMaker.
+* Score the model using sample scoring records and the scoring endpoint.
+* Setup AI OpenScale Data Mart
+* Bind the SageMaker model to the AIOS Data Mart
+* Add subscriptions to the Data Mart
+* Enable payload logging and performance monitor for both subscribed assets
+* Use Data Mart to access tables data via subscription
+
+![architecture](doc/source/images/architecture.png)
```
Comment (sanjeevghimire) [inline README.md]: this link is broken. but others LGTM

File: README.md (hunk 1 above)
Comment (markstur) [reply README.md]: also missing the dash after ai for the 3rd link to work

File: README.md (hunk 6 above)
Comment (markstur) [inline README.md]: missing png

**Pull request merged**